### PR TITLE
Add support for the `rows` attribute

### LINKED ISSUE #2752: [Textarea] Add support for the `rows` attribute
The **textarea** Angular component currently has a fixed height equivalent to three rows of text.

Would it be possible to let us add the `rows` attribute (via a `[rows]` input for instance), and let the textarea natively size itself based on its value instead of a fixed height?

Alternatively, we could use a custom property instead of a fixed value to set the height of the textarea:

```
--components-textfield-rows: 3;

  .textField-input-value:is(textarea) {
    height: calc(var(--components-textfield-rows) * 1lh + var(--component-textField-padding) * 2);
  }
```

### PR DESCRIPTION
**Pull request by vvalentin-lucca**
## Description

Fix #2752 

-----

-----

### CODE REVIEW

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/16975/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2788/storybook

**Review by jeremie-lucca — APPROVED** [review]

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/17006/ui-diff)

File: packages/ng/forms/textarea-input/textarea-input.component.ts [hunk 1]
```diff
@@ -18,4 +18,7 @@ export class TextareaInputComponent {
 
 	@Input()
 	placeholder: string = '';
+
+	@Input()
+	rows?: number;
```
Comment (shajz) [inline packages/ng/forms/textarea-input/textarea-input.component.ts:23]: Should we set 2 as the default value? It's the official default value.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea#rows
```suggestion
	rows: number = 2;
```

Comment (vvalentin-lucca) [reply packages/ng/forms/textarea-input/textarea-input.component.ts:23]: Good point. 

The idea was to respect the design, but the question also existed of modifying it.

Comment (vvalentin-lucca) [reply packages/ng/forms/textarea-input/textarea-input.component.ts:23]: Good point.
The idea was to respect the design, but the question also existed of modifying it.

Does this make the parameter mandatory? I think not.

Comment (shajz) [reply packages/ng/forms/textarea-input/textarea-input.component.ts:23]: No, if the parameter is not provided, the default value will be used.

Comment (vvalentin-lucca) [reply packages/ng/forms/textarea-input/textarea-input.component.ts:23]: On the design side, the intention is to stay with 3 lines by default, so we'll keep it that way.

**Review by Supamiu — APPROVED** [review]

**Pull request merged**